packages/popover/src/index.tsx

```diff
@@ -241,13 +262,35 @@ export const Popover: React.FC<PopoverProps> = ({
     },
     [close]
   )
+  const elevateTarget = () => {
```

Comment:
Not quite sure if we should elevate popover target as well

Reply:
I'm not sure either. Maybe make it a component prop and also add to theme defaults